jni/CMakeLists.txt

```diff
@@ -69,6 +69,12 @@ install(TARGETS ${KNN_INDEX}
         COMPONENT library)
 
 # CPack section to build artifacts
+if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL aarch64)
+    set(MACH_ARCH arm64)
+elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL x86_64)
+    set(MACH_ARCH x64)
+endif()
+
 set(KNN_MAINTAINER "OpenDistro for Elasticsearch Team <opendistro@amazon.com>")
 set(ODFE_DOWNLOAD_URL "https://opendistro.github.io/elasticsearch/downloads")
```

Comment:
FAR: ODFE_DOWNLOAD_URL should point to https://opendistro.github.io/for-elasticsearch/downloads.html

Reply:
Good catch. Will fix.